wp-content/themes/core/content/single/post.twig

```diff
@@ -1,32 +1,50 @@
 <article class="item-single">
 
-	{{ post.featured_image }}
-
-	<div class="item-single__content t-content">
-		{{ post.content }}
-	</div>
-
-	<footer class="item-single__footer">
-
-		<ul class="item-single__meta">
-
-			<li class="item-single__meta-date">
+	<div class="item-single__featured-image">{{ post.featured_image }}</div>
+
+	<div class="l-container__stagger item-single__content">
+		<h1 class="item-single__headline">{{ post.title }}</h1>
+		<div class="item-single__meta">
```

Comment:
What do you think about updating this wrapper to be `<ul>` (and it's children accordingly) given it's a list of meta?